build.gradle

```diff
@@ -30,27 +31,61 @@ repositories {
 
 dependencyManagement {
     imports {
-        mavenBom "org.camunda.bpm:camunda-bom:${camunda_version}"
-        mavenBom "org.springframework.boot:spring-boot-starter-parent:${spring_boot_version}"
+        mavenBom "org.camunda.bpm:camunda-bom:${camundaVersion}"
+        mavenBom "org.springframework.boot:spring-boot-starter-parent:${springBootVersion}"
     }
 }
 
 dependencies {
-    compileOnly("org.projectlombok:lombok:${lombok_version}")
-    annotationProcessor("org.projectlombok:lombok:${lombok_version}")
-
-    implementation("org.camunda.bpm.springboot:camunda-bpm-spring-boot-starter-external-task-client:${camunda_version}")
-    implementation("org.camunda.connect:camunda-connect-http-client:${camunda_http_client_version}")
+    annotationProcessor('org.projectlombok:lombok')
+    compileOnly('org.projectlombok:lombok')
 
-    // workaround for external task dependency issue - https://github.com/camunda/camunda-bpm-platform/issues/3829
-    implementation("jakarta.xml.bind:jakarta.xml.bind-api:${jakarta_xml_bind_api_version}")
```

Comment:
@bschlosser Whilst the code builds it no longer runs without this dependency as it throws class not found exceptions. So I am going to restore this back for now as I need to get this working ASAP for the client.